### ISSUE TITLE
[nri-bundle] Warnings fired when installing the chart

### ISSUE THREAD

**Issue by paologallinaharbur** [newrelic/helm-charts#304]
**Bug description**

When installing some warnings are fired complaining about the deprecated api version of some of the resources.

This prevents some users to use the solution. Most of the resources have already updated the manifests to use the new api version. We should do the same.

```
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding
Warning: admissionregistration.k8s.io/v1beta1 MutatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 MutatingWebhookConfiguration
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRole is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRole
```

Examples of files affected:
 - nri-bundle/charts/nri-metadata-injection/templates/mutationwebhookconfiguration.yaml
 - nri-bundle/charts/kube-state-metrics/templates/clusterrolebinding.yaml (**This is a dependency**)
 - ri-bundle/charts/kube-state-metrics/templates/clusterrole.yaml (**This is a dependency**)

**Comment by paologallinaharbur** [comment]
`rbac.authorization.k8s.io/v1 should be available since v1`
`admissionregistration.k8s.io/v1 should be available since 1.16`

Ensure that the Kubernetes cluster is at least as new as v1.16 (to use admissionregistration.k8s.io/v1), or v1.9 (to use admissionregistration.k8s.io/v1beta1).

**Comment by paologallinaharbur** [comment]
The first one should be safe to be changed, the admissionregistration one could cause user with old versions of k8s to not be able to upgrade
